docs/references/general/dat.md

```diff
@@ -0,0 +1,113 @@
+| *txpl*                 | *us*      | short     | Transmitted pulse length                                                                                                                                                                         |
+| *mpinc*                | *us*      | short     | Multi-pulse increment (lag separation)                                                                                                                                                           |
+| *mppul*                | *None*    | short     | Number of pulses in the sequence                                                                                                                                                                 |
+| *mplgs*                | *None*    | short     | Number of lags in the lag table                                                                                                                                                                  |
+| *nrang*                | *None*    | short     | Number of range gates                                                                                                                                                                            |
+| *frang*                | *km*      | short     | Distance to the first range gate                                                                                                                                                                 |
+| *rsep*                 | *km*      | short     | Range separation                                                                                                                                                                                 |
+| *xcf*                  | *None*    | short     | XCF flag, indicating whether the power, velocity, spectral width and phi0 parameters fitted from the cross-correlation function are available (xcf=1) or not available (xcf=0)                   |
+| *tfreq*                | *kHz*     | short     | Transmitted frequency                                                                                                                                                                            |
+| *mxpwr*                | *None*      | int       | Maximum power                                                                                                                                                                                    |
+| *lvmax*                | *None*    | int       | Maximum noise level allowed                                                                                                                                                                      |
+| usr_resL2              | *None*    | int       | User defined variable                                                                                                                                                                            |
```

Comment:
``` suggestion
| usr_resL1 | *None* | int | offset | 
| usr_resL2              | *None*    | int       | User defined variable                                                                                                                                                                          |
```

Reply:
@mts299 since we're unsure about the usage of `offset` here, I edited your suggestion to "user defined variable".